tests/parser/functions/test_slice.py

```diff
@@ -31,7 +31,7 @@ def test_test_slice2(get_contract_with_gas_estimation):
 def slice_tower_test(inp1: bytes[50]) -> bytes[50]:
     inp: bytes[50] = inp1
     for i in range(1, 11):
-        inp = slice(inp, 1, 30 - i * 2)
+        inp = slice(inp, 1, convert(30 - i * 2, uint256))
```

Comment:
The convert shouldn't be necessary, what's going wrong in this statement?

Reply:
It's because logic in `parser` still implicitly casts `i` as an `int128`.  In the future the `convert` won't be necessary but today it is :(

Reply:
Okay gotcha, leave a TODO for that?